src/legacy/legacy-default-xrp-client.ts

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable @typescript-eslint/require-await */
```

Comment:
good Q.  these are here for the paymentHistory method in the legacy client.  Since the legacy client is no longer being supported, this method will not be implemented here and instead just throws an unimplemented error (but it has to be here to conform to the interface).  I actually moved the disable `no unused vars` down to be right before this method, and got rid of the async keyword entirely (and the corresponding linter disabling) because I realized it's not required.  good catch!